package.json

```diff
@@ -47,6 +47,7 @@
     "file-loader": "^0.9.0",
     "highlight.js": "~9.14.0",
     "node-sass": "^4.5.0",
+    "optimize-css-assets-webpack-plugin": "^3.2.0",
```

Comment:
Created a packaging PR - https://github.com/theforeman/foreman-packaging/pull/5980

Reply:
Merged

Reply:
Why we haven't included it in the vendor/builder only? :thinking: